src/main/java/seedu/address/model/Model.java

```diff
@@ -81,6 +83,14 @@
      */
     void setPatient(Patient target, Patient editedPatient);
 
+    /**
+     * Checks if patient is already assigned to a room.
+     *
+     * @param name Of the patient.
+     * @return Boolean value of whether patient is already assigned.
+     */
+    boolean isPatientAssignedRoom(Name name);
```

Comment:
Just small gramma "AssignedToRoom"

Reply:
HAHAHAH, i wrote it that way so the method name not so long. will change it later!